### PR TITLE
Update pipeline.yml for new Artifactory url

### PR DESCRIPTION
**Pull request by jesusHCG**
### Description
Change of Artifactory url and docker image

### Related GitHub issue or JIRA story
https://jira.autodesk.com/browse/DYN-2071
https://jira.autodesk.com/browse/DYN-2072

### Checklist
- [ ] PR title reflects the changes it introduces. Consider using this template: QNTM-XXXX: Implement X Feature
- [ - ] Code compiles correctly
- [ - ] No linting errors
- [ - ] All tests passing
- [ ] Updated README.md / documentation (if necessary)

### CODE REVIEW

File: pipeline.yml [hunk 1]
```diff
@@ -3,27 +3,27 @@ env:
...
 nodejs_version: "10"
 
 build:
   customized_builder_image:
     registry: artifactory
-    url: autodesk-docker.art-bobcat.autodesk.com/quantum/hardened-nodejs:latest
+    url: artifactory.dev.adskengineer.net/autodeskcloud/cloudosv2-base-nodejs-10:latest
   before_scripts:
-    - "npm install"
+    - "sudo npm install --unsafe-perm=true"
```
Comment (QilongTang) [inline pipeline.yml]: Why do we need to perform this change here?

Comment (jesusHCG) [reply pipeline.yml]: We are having difficulties executing the npm install inside the docker container.

image

We find this solution 

https://stackoverflow.com/questions/35954725/error-eacces-permission-denied-when-trying-to-install-eslint-using-npm

Or reinstall npm

https://docs.npmjs.com/resolving-eacces-permissions-errors-when-installing-packages-globally

Comment (mjkkirschner) [reply pipeline.yml]: @pinzart @jasonstratton  should this be necessary to install npm?

Comment (pinzart90) [reply pipeline.yml]: I commented on slack regarding this issue

Comment (jesusHCG) [reply pipeline.yml]: We already removed the sudo instruction.

**Comment by jesusHCG** [comment]
Hi @QilongTang, @mjkkirschner and @pinzart90, We already have a new docker image, can you help us validate this pull request to do some test?

**Comment by QilongTang** [comment]
@jesusHCG I am going to merge and request you test on the VM if that is OK, let me know!

File: pipeline.yml [hunk 2]
```diff
@@ -3,18 +3,18 @@ env:
...
 pipeline_os: "Linux"
 force_slack_notification : true
-language: typescript
 nodejs_version: "10"
 
 build:
   customized_builder_image:
     registry: artifactory
-    url: autodesk-docker.art-bobcat.autodesk.com/quantum/hardened-nodejs:latest
+    url: artifactory.dev.adskengineer.net/docker-local-v2/docker-local-v2/dynamo/dynamolibrariejs:latest
```
Comment (pinzart90) [inline pipeline.yml]: What's up with the path to the image ?
`/docker-local-v2/docker-local-v2` 
@jesusHCG @mjkkirschner  ?
seems like this works too `artifactory.dev.adskengineer.net/docker-local-v2/dynamo/dynamolibrariej`

Comment (jesusHCG) [reply pipeline.yml]: i used the url from the artifactory

image

**Pull request merged**